tests/operations/bitwise/RandomFill.cpp

```diff
@@ -0,0 +1,52 @@
+#define CATCH_CONFIG_MAIN
+#include "Catch/single_include/catch2/catch.hpp"
+
+#include "conduit/include/uitsl/polyfill/bit_cast.hpp"
+
+#include "Empirical/include/emp/data/DataNode.hpp"
+
+#include "sgpl/algorithm/execute_cpu.hpp"
+#include "sgpl/hardware/Core.hpp"
+#include "sgpl/operations/bitwise/RandomFill.hpp"
+#include "sgpl/program/Program.hpp"
+#include "sgpl/spec/Spec.hpp"
+#include "sgpl/utility/ThreadLocalRandom.hpp"
+
+using spec_t = sgpl::Spec<sgpl::OpLibrary<sgpl::RandomFill>>;
+
+TEST_CASE("Test RandomFill") {
```

Comment:
This test case is a bit more complicated. Give a high level comment describing what we’re doing